docs/pages/web/sheetmobile.js

```diff
@@ -327,6 +328,27 @@ When using these render props, just pass the argument \`onDismissStart\` to your
             }
           />
         </MainSection.Subsection>
+
+        <MainSection.Subsection
+          title="External handlers"
+          description={`SheetMobile consumes external handlers from GlobalEventsHandlerProvider.
+
+Handlers:
+- onOpen: executed on useEffect
```

Comment:
"executed on useEffect" doesn't really mean anything. An effect will get run once when the component mounts, but then also again any time its dependencies change. I wouldn't get caught up on the use of `useEffect` here — that's really an implementation detail. What's pertinent to the dev is that `onOpen` will run when the sheet opens, and that `onClose` will run just before the sheet closes. Whether it uses `useEffect` or some other mechanism is irrelevant — and really, if the developer needs to be be aware of that mechanism, we have a leaky abstraction.